OpenBudgeteer.Blazor/Program.cs

```diff
@@ -1,19 +1,32 @@
+using System;
+using System.Linq;
 using Microsoft.AspNetCore.Hosting;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using OpenBudgeteer.Data;
 
 namespace OpenBudgeteer.Blazor;
 
 public class Program
 {
     public static void Main(string[] args)
     {
+        AppContext.SetSwitch("Npgsql.EnableLegacyTimestampBehavior", true);
         CreateHostBuilder(args).Build().Run();
     }
 
-    public static IHostBuilder CreateHostBuilder(string[] args) =>
+    private static IHostBuilder CreateHostBuilder(string[] args) =>
         Host.CreateDefaultBuilder(args)
```

Comment:
For `"CONNECTION_PROVIDER": "POSTGRES"` I get below exception here:

```
42501: permission denied for schema public
```

DB and User creation was working fine. Below I share my Postgres Docker compose and the config settings:

``` yaml
version: '3.1'

services:

  db:
    image: postgres
    restart: always
    ports:
      - 5432:5432
    environment:
      POSTGRES_PASSWORD: example
      
  adminer:
    image: adminer
    restart: always
    ports:
      - 8081:8080
```

``` json
"CONNECTION_PROVIDER": "POSTGRES",
  "CONNECTION_DATABASE": "openbudgeteer_dev",
  "CONNECTION_SERVER": "192.168.178.201",
  "CONNECTION_PORT": "5432",
  "CONNECTION_USER": "openbudgeteer_dev",
  "CONNECTION_PASSWORD": "openbudgeteer_dev",
  "CONNECTION_ROOT_PASSWORD": "example",
```